redis/redis/broker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM redis:latest
+
+COPY redis.conf /usr/local/etc/redis/redis.conf
```

Comment:
Yes, there was a reason for it.

The main reason was that I wanted to see the redis logs. Default writes to stdout. This is not ok in my opinion, if you want to examine these logs later.